ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-release-4.20__periodics.yaml

```diff
@@ -169,6 +183,31 @@ tests:
       TEST_SUITE: openshift/machine-config-operator/disruptive
     workflow: baremetalds-e2e-ovn-dualstack
   timeout: 6h0m0s
+- as: e2e-aws-ovn-ocl
+  cron: 0 8 * * 1
```

Comment:
nit: we probably can just use `interval: 168h` if we want to run this weekly, so we don't have to specify an exact time/day to run (less of an issue for AWS jobs, more of an issue for metal jobs since we can run out of quota there), but is fine as is as well